### PR TITLE
Challenge 0

### PR DESCRIPTION
**Pull request by Thuku**

### CODE REVIEW

File: ui/dashboard.html [hunk 1]
```diff
@@ -54,7 +54,40 @@
                 </div>
 
                 <div class="col s9">
```
Comment (EinsteinCarrey) [inline ui/dashboard.html]: I love the way you have indented your code. Easily readable and user-friendly

File: ui/create.html [hunk 2]
```diff
@@ -54,7 +54,49 @@
...
+                    </p>
+                    <p>
+                        <table>
+        <thead>
+          <tr>
+              <th>ID</th>
+              <th>Activity Name</th>
+          </tr>
+        </thead>
+
```
Comment (EinsteinCarrey) [inline ui/create.html]: Consider indenting this code ( **Make it more readable** )

**Comment by EinsteinCarrey** [comment]
I would recommend that you finish working on a **feature** before committing.

**Comment by Thuku** [comment]
@EinsteinCarrey I will improve onthat

**Comment by brayoh** [comment]
I think It's looking great

**Pull request merged**